src/components/modal/modal.js

```diff
@@ -39,7 +39,11 @@ class Modal extends PureComponent {
     if (closeOnEsc && doc) {
       doc.removeEventListener('keydown', this.handleKeydown);
     }
-    this.portalElement.remove();
+    // check if the portalElement is a thing
+    if (this.portalElement) {
```

Comment:
This verification is not need it, `this.portalElement` will be always a html (its set on the componentDidMount`) adding this verification that will be always true reduce the test coverage and there is no way to correctly create a test where this is false.

Reply:
Done :)